src/System.Net.Primitives/tests/FunctionalTests/CookieTest.cs

```diff
@@ -337,5 +337,26 @@ public static void ToString_Compare_Success()
             c.Port = "";
             Assert.Equal("name=value; $Path=path; $Domain=domain; $Port", c.ToString());
         }
+        
+        [Theory]
+        [MemberData(nameof(HashCodesTestData))]
+        public static void HashCodes_ShouldWorkLikeTuple(string name, string value, string path, string domain, int version)
```

Comment:
I would just delete the test (or modify it to call GetHashCode but ignore the result to ensure it doesn't throw).  If we don't want to codify the behavior of GetHashCode, let's not write the test that asserts stuff.